src/composables/useColourScale.ts

```diff
@@ -55,7 +55,7 @@ export const useColourScale = (selectedIndicators: Ref<Dict<FeatureIndicatorValu
 
     const getColour = (indicator: string, featureIndicators: FeatureIndicatorValues | undefined) => {
         // If indicators do not exist for this feature, return transparent
-        if (!featureIndicators) {
+        if (!featureIndicators || !featureIndicators[indicator]) {
```

Comment:
checking - if `featureIndicators` _is_ present and `featureIndicators[indicator]` _isn't_, we should return the 0-colour?